docs/examples/basic_conceptual_guide/Part_1-model_deployment/Readme.md

```diff
@@ -0,0 +1,293 @@
+```
+<model-repository>/
+  <model-name>/
+    [config.pbtxt]
+    [<output-labels-file> ...]
+    <version>/
+      <model-definition-file>
+    <version>/
+      <model-definition-file>
+    ...
+  <model-name>/
+    [config.pbtxt]
+    [<output-labels-file> ...]
+    <version>/
+      <model-definition-file>
+    <version>/
+      <model-definition-file>
+    ...
+  ...
+```
+
+There are three important components to be discussed from the above structure:
+* `model name`: The identifying name for the model.
+* `config.pbtxt`: For each model, users can define a model configuration. This configuration, at minimum, needs to define: the platform and the name, shape, and datatype of model inputs and outputs. For most of the popular backends, this configuration file is autogenerated with defaults. Explore the [common protobuf](https://github.com/triton-inference-server/common/blob/main/protobuf/model_config.proto) for the full specification of the configuration file.
+* `version`: versioning makes multiple versions of the same model available for use depending on the policy selected. [More Information about versioning.](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_configuration.md#version-policy)
+
+For this example the model repository structure would need to be set up in the following manner:
+```
+model_repository
+|
+|-- text_detection
+|   |
+|   |-- config.pbtxt
+|   |-- 1
+|       |
+|       |-- model.onnx
+|
+|-- text_recognition
+    |
+    |-- config.pbtxt
+    |-- 1
+        |
+        |-- model.onnx
+```
+
+With the models and the filestructure ready, the only point of discussion remaining are the `config.pbtxt` files. As an example, consider the model configuration for the `EAST text detection` model. This is an ONNX model that has one `input` and two `output` layers.
+
+```
+name: "text_detection"
+platform: "onnxruntime_onnx"
+max_batch_size : 256
+input [
+  {
+    name: "input_images:0"
+    data_type: TYPE_FP32
+    dims: [ -1, -1, -1, 3 ]
+  }
+]
+output [
+  {
+    name: "feature_fusion/Conv_7/Sigmoid:0"
+    data_type: TYPE_FP32
+    dims: [ -1, -1, -1, 1 ]
+  }
+]
+output [
+  {
+    name: "feature_fusion/concat_3:0"
+    data_type: TYPE_FP32
+    dims: [ -1, -1, -1, 5 ]
+  }
+]
+```
+* **name:** "name" is an optional field, the value of which should match the name of the directory of the model.
+* **platform:** This field indicates which framework is being used to run the model. Triton supports a wide variety of backends like TensorFlow, Pytorch, Python, ONNX and more. For a complete list of field selection refer to [these comments](https://github.com/triton-inference-server/common/blob/main/protobuf/model_config.proto#L1791)
```

Comment:
Use `backend` instead of `platform` everywhere in these docs. Also update the names to match. Ex: `tensorrt` not `tensorrt_plan`, etc.